lib/processors/bundlers/moduleBundler.js

```diff
@@ -112,16 +112,28 @@ class LocatorResourcePool extends ResourcePool {
  * @param {boolean} [parameters.options.bundleOptions.addTryCatchRestartWrapper=false] Whether to wrap bootable module bundles with a try/catch to filter out "Restart" errors
  * @param {boolean} [parameters.options.bundleOptions.usePredefineCalls=false] If set to 'true', sap.ui.predefine is used for UI5 modules
  * @param {number} [parameters.options.bundleOptions.numberOfParts=1] The number of parts the module bundle should be splitted
+ * @param {number} [parameters.options.bundleOptions.ignoreMissingModules] TODO name
  * @returns {Promise<module:@ui5/fs.Resource[]>} Promise resolving with module bundle resources
  */
 module.exports = function({resources, options}) {
 //	console.log("preloadBundler bundleDefinition:");
 //	console.log(JSON.stringify(options.bundleDefinition, null, 4));
 
-	const pool = new LocatorResourcePool();
+	const bundleOptions = options.bundleOptions || {};
+
+	// Apply default bundleOptions
+	if (bundleOptions.optimize === undefined) {
```

Comment:
I'm not completely sure, but I think the default handling is now different.

`optimize` will now always be set to true. Previously it has been set to `true` only in case no `bundleOptions` were provided at all.

Reply:
Done in latest commit